app/herdbook.py

```diff
@@ -230,41 +233,102 @@ def herd(h_id):
     return jsonify(data)
 
 
-@APP.route("/api/individual/<i_id>")
+@APP.route("/api/individual/<i_number>")
```

Comment:
This will breake stuff I think since the ind = da.get_individual(i_id, user_id)
is based on the internal database ID and not the numbers 

What do you say?
@norling

Reply:
no this uses the i_number so this is more clear.